ncov-recombinant/0.6.1/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mambaorg/micromamba:0.24.0 as app
+
+USER root
+
+WORKDIR /
+
+#metadata
+LABEL base.image="FROM mambaorg/micromamba:0.24.0"
+LABEL dockerfile.version="1"
+LABEL software="ncov-recombinant"
+LABEL software.version="v0.6.1"
+LABEL description="SARS-CoV-2 recombinant sequence detection"
+LABEL website="https://github.com/ktmeaton/ncov-recombinant"
+LABEL license="https://github.com/ktmeaton/ncov-recombinant/blob/master/LICENSE"
+LABEL maintainer="Wes Hottel"
+LABEL maintainer.email="wesley-hottel@uiowa.edu"
+
+#Install dependencies
+RUN apt-get update && apt-get install -y python \ 
+  git \
+  wget && \
+  rm -rf /var/lib/apt/lists/* && apt-get autoclean
```

Comment:
We have found that images are smaller when using  `--no-install-recommends`

Could you adjust line 19 to 
```
apt-get update && apt-get install --no-install-recommends -y python \
```